pandas/tests/formats/test_format.py

```diff
@@ -3938,6 +3938,15 @@ def test_period(self):
         self.assertEqual(str(df), exp)
 
 
```

Comment:
Was gonna do fixtures as a follow up. Right now it's runnable under nose and pytest.